tests/std/tests/GH_002989_nothrow_unwrappable/test.cpp

```diff
@@ -117,4 +118,6 @@ int main() {
 #if _HAS_CXX17
     do_full_test<path::iterator, false>();
 #endif
+#endif

```

Comment:
This should be commented because it's far away from the `#if 0`. Also, why is this disabled?

Reply:
ah, oops, testing that didn't get removed.